Move prover returns: exiting with boogie verification errors
error: unexpected boogie output: `Prover error: Unexpected prover response getting model: (() (define-fu ..`

   ┌── <unknown>:1:1 ───
   │
 1 │ <unknown>
   │ ^^^^^^^^^
   │

error: post-condition does not hold

    ┌── tests/sources/functional/address_quant.move:53:10 ───
    │
 53 │          invariant atMostOne();
    │          ^^^^^^^^^^^^^^^^^^^^^^
    │
    =     at tests/sources/functional/address_quant.move:46: multiple_copy_incorrect
    =     at tests/sources/functional/address_quant.move:53
    =     at tests/sources/functional/address_quant.move:54
    =     at tests/sources/functional/address_quant.move:46: multiple_copy_incorrect
    =     at tests/sources/functional/address_quant.move:47: multiple_copy_incorrect
    =     at tests/sources/functional/address_quant.move:48: multiple_copy_incorrect
    =     at tests/sources/functional/address_quant.move:53
